src/config/cambodia/prism.json

```diff
@@ -1,6 +1,7 @@
 {
   "country": "Cambodia",
   "alertFormActive": false,
+  "navigationDropdown": false,
```

Comment:
@wadhwamatic should the default be false and we only activate it for specific countries? Or the opposite?
Not sure we need to set it explicitly in the config for each counyty. What do you think?

Reply:
agreed with @ericboucher - let's remove the flag and make the default behavior false. we would specify true when required for a deployments. thanks for implementing and letting us know when ready @JorgeMartinezG

Reply:
Fix included in 7eae209b4048ba6f4869f4732b13ff75ee0806de

Reply:
@wadhwamatic fixed in ad4a5d5c780cbad40d4aa1fc5bd4f2cf09090997